trade-finance-manager-ui/templates/case/parties/parties.njk

```diff
@@ -10,6 +10,10 @@
 {% extends "case/case.njk" %}
 {%- from "govuk/components/tag/macro.njk" import govukTag -%}
 
+{% block pageTitle -%}
+  Deal {{ deal.details.ukefDealId }} - Parties
```

Comment:
GEF deals?

Reply:
Yes, it works for GEF, other deals, and all Facilities.